scapy/utils.py

```diff
@@ -446,6 +454,43 @@ def atol(x):
     return struct.unpack("!I", ip)[0]
 
 
+def valid_ip(addr):
+    addr = plain_str(addr)
+    try:
+        atol(addr)
+    except:
```

Comment:
Same comment for except. This also applies to the others try/except blocks.

Reply:
You're totally right. I'll have a look.